src/sentry/static/sentry/app/components/assistant/guideAnchor.jsx

```diff
@@ -64,7 +64,11 @@ const GuideAnchor = createReactClass({
     let {target, type} = this.props;
 
     return (
-      <GuideAnchorContainer innerRef={el => (this.anchorElement = el)} type={type}>
+      <GuideAnchorContainer
+        className={this.props.className}
```

Comment:
Do you need the className property, I don't think it's ever passed, nor is it in the proptypes defined for this component

Reply:
hmm this looks like it was leftover from a pervious iteration- thanks for the catch